bin/summary_busco.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+#USAGE: ./summary.busco.py *.txt
+
+import re
+from sys import argv
+
+#"# Summarized benchmarking in BUSCO notation for file MEGAHIT-testset1.contigs.fa"
+#"	C:0.0%[S:0.0%,D:0.0%],F:0.0%,M:100.0%,n:148"
+
+regexes = {
```

Comment:
Is this meant to be here? Looks like it's from the software versions script, and is immediately overwritten..